lib/charms/opensearch/v0/opensearch_exceptions.py

```diff
@@ -92,3 +92,11 @@ class OpenSearchScaleDownError(OpenSearchError):
 
 class OpenSearchIndexError(OpenSearchError):
     """Exception thrown when an opensearch index is invalid."""
+
+
+class OpenSearchPluginError(OpenSearchError):
```

Comment:
Could we put this class in `opensearch_plugins.py`?

Reply:
We should move it to opensearch_plugin.py and opensearch_keystore.py